ServerHttp/src/main/kotlin/io/raspberrywallet/mock/Base58.java

```diff
@@ -0,0 +1,61 @@
+package io.raspberrywallet.mock;
+
+import java.util.Arrays;
+
+/**
+ * For mocking address generation
+ */
+class Base58 {
+    public static final char[] ALPHABET = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz".toCharArray();
```

Comment:
Missing 0

Reply:
Base58 is Base64, but without confusing chars like 0(zero) and O(capital o), I(capital i) and l(lowwer L) and without non-alpha chars, that is  + and /
So there is no bug